packages/client/src/components/Location/LocationFinder.tsx

```diff
@@ -147,14 +148,15 @@ const LocationFinder: React.FC<LocationFinderProps & MatomoTrackEventProps> = ({
 
   // GraphQL error
   useEffect(() => {
-    if (graphqlError) {
+    if (graphqlError && currentError !== graphqlError?.message) {
```

Comment:
Nice update. However, if the user now clears the input fields the Alert will be removed. If the user now makes a new input, the Alert will be showed again and the event will be triggered again.

What about this?  Replace `[currentError, setCurrentError]` with `[hasError, SetError]`. And show the alert permanently and fully disable a new event being triggered? Maybe we should also disable the submit in that case.

Reply:
Yes,  i noticed this. But i figured to keep the same way the api down alert is shown. 

But let's change to alert too. It makes no sense to keep it hiding and showing when the api is probably still down.

Reply:
exactly! nice